src/components/eventSingleComponent/eventSingleComponent.scss

```diff
@@ -0,0 +1,23 @@
+.kd-layout__object-fit--cover {
```

Comment:
Don't redeclare this just include layout.scss from the foundation

@use '~@kyndryl-design-system/shidoka-foundation/scss/utility/foundation.scss';